### PR TITLE
Sync repos for the release 170.3.0

### PR DESCRIPTION
**Pull request by ssreerama**

### CODE REVIEW

File: .azuredevops/policies/approvercountpolicy.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: approver_count
```
Comment (zijchen) [inline .azuredevops/policies/approvercountpolicy.yml:1]: I don't think we need this file in public repo

**Review by zijchen — APPROVED** [review]

File: .azuredevops/policies/approvercountpolicy.yml (hunk 1 above)
Comment (llali) [reply .azuredevops/policies/approvercountpolicy.yml:1]: correct, please remove

**Review by llali — CHANGES_REQUESTED** [review]
please also port the commit for the release note

File: .azuredevops/policies/approvercountpolicy.yml (hunk 1 above)
Comment (ssreerama) [reply .azuredevops/policies/approvercountpolicy.yml]: Done! I removed all other onebranch template file changes, unnoticed this one, thanks for noticing it.

**Review by llali — APPROVED** [review]

**Review by llali — APPROVED** [review]

**Pull request merged**